tools/coredumper/sof-coredump-reader.py

```diff
@@ -690,9 +734,17 @@ def __init__(self, args):
 			stdoutPrint("set *0x{:08x}=0x{:08x}\n"
 				.format(addr, dw))
 
+		if self.core_dump.exccause:
+			verbosePrint("# exccause: " + EXCCAUSE_CODE[self.core_dump.exccause][0])
```

Comment:
A header like '\*EXCEPTION\*' would be a nice way to get an attention and highlight the next part.

Reply:
Done